app.py

```diff
@@ -189,13 +189,8 @@ def get_pages_details(links):
 # ===
 @app.route('/')
 def homepage():
-    return flask.render_template('index.html', now=datetime.datetime.utcnow())
-
-
-@app.route('/index2')
-def new_homepage():
     return flask.render_template(
-        'index2.html',
+        'index2.html',  # old home page is in index.html in case we need it
```

Comment:
Please remove `index2.html` and put this at `index.html` and remove this comment. We have version control if we need to retrieve anything.